src/commands/profiles/port.js

```diff
@@ -0,0 +1,83 @@
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const { cli } = require('cli-ux');
+
+class ProfilesPort extends BaseCommand {
+  async run() {
+    await super.run();
+
+    const query = await this.inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'confirmation',
+        message:
+          `This command will port profile(s) from keytar` +
+          ` to config file at location ${this.configFile.filePath}. Continue?`,
+        default: false,
+      },
+    ]);
+
+    if (!query.confirmation) {
+      this.logger.warn('Abort!');
+      return;
+    }
+
+    const { projects } = this.userConfig;
+
+    if (this.args.profile) {
+      const sanitizedProfileId = this.args.profile.trim();
+      const project = projects.find((p) => p.id === sanitizedProfileId);
+
+      if (!project) {
+        this.logger.error(
+          `Unable to port profile ${sanitizedProfileId} since it doesn't exist or has already been ported!`,
+        );
+        return;
+      }
+
+      await this.portProfile(project);
+    } else {
+      if (projects.length === 0) {
+        this.logger.info('All profiles already ported to config file!');
+        return;
+      }
+
+      for (const project of projects) {
+        await this.portProfile(project);
+      }
+
+      this.logger.info('Profiles port process complete!');
```

Comment:
Should we make this a hidden command after all projects have been ported?

Reply:
I guess I couldn't get this one..Could you please add details? 

It should only log when there's at least one profile to work on (failure or success case), and then indicate that the process is now complete towards the end. I might be interpreting the comment incorrectly...

Reply:
So this is a one-time operation, so users needn't use it once it's run successfully. So, after porting is complete, we make it a hidden command, so when the users run profiles help, this port won't show up as one subcommand. It can become a hidden command, kind of like the Zork one

Reply:
If we mark the command as hidden after this process, it wouldn't be saved anywhere on config, and would remain just for that execution context. 
The next time user triggers `profiles --help` it'd check `ProfilesPort.hidden` flag in the newly loaded context, where it's still marked as **false** , and would still show up. 

Please let me know if I still interpretted it wrongly, and we just needed to mark the command as `hidden` right away!

Reply:
That's fine, I think we can simply disable it when we are completely removing the keytar dependency later on once we see from user data that no calls are made with keys from keychain